tests/components/orderentry/OrdersTable.test.jsx

```diff
@@ -199,22 +211,31 @@ describe('getUUID() method', () => {
   });
 });
 
-describe('setDiscontinuedDrugOrder method', () => {
-  it('should call setDiscontinuedDrugOrder', async (done) => {
+describe('setDiscontinuedOrder method for drug orders', () => {
+  it('should call setDiscontinuedOrder for drug orders', async (done) => {
+    const renderedComponent = getComponent().instance();
+    sinon.spy(renderedComponent, 'setDiscontinuedOrder');
+    renderedComponent.setDiscontinuedOrder(mockDrugOrder);
+    expect(renderedComponent.setDiscontinuedOrder.calledOnce).toEqual(true);
+    done();
+  });
+});
```

Comment:
I think it's been agreed that the spy methods would no longer be used on this project...

Reply:
Yeah, I remember I saw a PR raised by you to fix this so I assumed these test cases are still like this because you saw them fit to be like that, I just edited them and passed in new props.
Any reason why you left them like that @Efosaok because the PR for that story has been merged.

Reply:
then that means it must have been missed out in my PR, but as part of the codebase cleanup we are trying to do, we've agreed to not use spy methods, so create another ticket for this or if it's just this file, you can just address it on this PR.

Reply:
I will create another ticket to fix this along with some minor enhancements I discovered in the application @Efosaok